src/ResourceManager/Network/Commands.Network/Microsoft.Azure.Commands.Network.format.ps1xml

```diff
@@ -2,6 +2,32 @@
 <Configuration>
   <ViewDefinitions>
```

Comment:
Added autogeneration of this code:
If nothing is specified in config for psView, generate an item for each property
Else, add code for each item in config. User can add any value there, so in case of custom values (i.e. name is out of existing object properties list) they have to create custom description in partial class.